src/material/tabs/tab-nav-bar/tab-nav-bar.spec.ts

```diff
@@ -545,3 +573,23 @@ class TabLinkWithNgIf {
 class TabBarWithInactiveTabsOnInit {
   tabs = [0, 1, 2];
 }
+
+@Component({
+  template: `
+    <nav [animationDuration]="500" mat-tab-nav-bar [tabPanel]="tabPanel">
+    <a
+      mat-tab-link
+      *ngFor="let link of links"
+      (click)="activeLink = link"
+      [active]="activeLink == link"
```

Comment:
You should be able to remove the `click` and `active` bindings since they aren't used in the test, as far as I can tell.